sources/models/src/vmware-k8s-1.25/defaults.d/26-aws-autoscaling.toml

```diff
@@ -0,0 +1 @@
+../../../shared-defaults/aws-autoscaling.toml
```

Comment:
We need to remove this from vmware-k8s. The autoscaling settings do not exist in vmware-k8s-1.25 variants.

Reply:
Done!